lab-bundle-spec.md

```diff
@@ -357,34 +364,58 @@ For a complete and up-to-date list of EC2 instance types, see [AWS official docu
 
 Specify which resource properties are given to the lab taker.
 
+For the lab taker to get access to a `gcp_project`, a `console_url` resource reference must be provided.
+For the lab taker to get access to an `aws_account`, one of the following resource references must be provided:
+- `console_url`
+- `sts_link`
+- `vnc_link`
+
+All `console_url`, `sts_link`, and `vnc_link` resource references are presented as a button to the lab taker. The label provided with the resource reference will be displayed on the button.
```

Comment:
Nit: "lab taker" -> "student". We have enough nomenclature as-is and I'd rather not introduce more unless we really need to.

Reply:
updated